xbmc/utils/Fanart.cpp

```diff
@@ -56,6 +55,21 @@ void CFanart::Pack()
   m_xml << fanart;
 }
 
+void CFanart::AddFanart(std::string image, std::string preview, std::string colors)
```

Comment:
If we're super strict here passing as a const ref can lead to worse performance. It will disable some optimization opportunities and the string is copied anyway because it's the only use of it. 

Const ref isn't necessarily a good default after c++11 because move semantics totally changed the rules. 

This method isn't called in a hot path so this comment is only for information, no changes requested